etc/vzlogger.conf

```diff
@@ -153,6 +155,27 @@
                 "middleware": "http://localhost/middleware.php"
             }
         },
+        {
+            /* A minimal example that "just works" on Debian. 
+               The steps needed are:
+               1. Install influxdb: sudo apt install influxdb influxdb-client
+               2. Start the client by calling influx on the command line and
+               create db: CREATE DATABASE vzlogger
+             */
+            "enabled": false,               // disabled meters will be ignored
+            "allowskip": false,                  // errors when opening meter may be ignored if enabled
+
+            "protocol": "random",
+            "interval": 2,
+            "max": 40.0,                    // has to be double!
+            "min": -5.0,                    // has to be double!
+            "channel": {
+                "uuid": "25a53b28-75ec-4764-a8a6-68587722ddab",
+                "api": "influxdb",
+                "identifier" : "1-0:16.7.0", // OBIS code for "power" 
```

Comment:
this will probably not work. i'd have to look up the exact mechanism.
the random meter oututs a single value that has the NilIdentifier:
https://github.com/volkszaehler/vzlogger/blob/master/src/protocols/MeterRandom.cpp#L90

also maybe have a look at the other options in
https://github.com/volkszaehler/vzlogger/blob/master/etc/vzlogger.conf.InfluxDB

Reply:
(also, currently all examples in vzlogger.conf use the volkszaehler api,
 and other apis have their own example.conf , maybe it should be kept that way.)

Reply:
It did work, at least I got values in the influx db. But I'll check again.

Reply:
I removed the unnecessary identifier.